e2e/cypress/e2e/order.cy.js

```diff
@@ -330,4 +330,12 @@ describe("Order", () => {
         .should("not.have.attr", "url");
     });
   });
+  describe("User has no agencies", () => {
```

Comment:
"If user logs in with MitID - and has no libraries associated with user account"